scripts/source-maintenance.sh

```diff
@@ -493,6 +493,27 @@ make -C src/http gperf-files
 
 run_ checkMakeNamedErrorDetails || exit 1
 
+# extract Author details from commit log
+collectAuthors ()
+{
+  tail -n +5 CONTRIBUTORS | sed -e 's/^[ ]*//' >AUTHORS.tmp
+
+  git log |
+    grep -E "^[ ]*[aA]uthor:" |
+    sed -e 's/^[ \t]*//' -e 's/[Aa]uthor:[ ]*//' -e 's/"//g' |
+    sed -e 's/[ ]*[,+&] /\n/g' >>AUTHORS.tmp
+
```

Comment:
At this point in the code we need to somewhat cleanup the AUTHORS.tmp list:
* drop all the Squid Project automation accounts (eg squidadm, root)
* de-duplicate core devs accounts from CVS
* rename authors (who have given permission for their details to be updated since initial commit was logged)

Reply:
Thank you for pushing this forward. And especially for picking up the ```Co-authored-by``` data source I missed.